### PR TITLE
Fix recursive config evaluation

### PR DESCRIPTION
**Pull request by vinjana**

### CODE REVIEW

**Comment by vinjana** [comment]
BatchEuphoria code change required to get this running.

File: RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy [hunk 1]
```diff
@@ -181,7 +180,7 @@ abstract class ExecutionService implements BEExecutionService {
         String cmdString
         if (!configurationDisallowsJobSubmission && !allJobsBlocked && !pidIsBlocked && !preventCalls && !isDummyCommand) {
             try {
-                cmdString = command.toString()
+                cmdString = command.toBashCommandString()
```
Comment (dankwart-de) [inline RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy]: I don't think it is appropriate to call it that way. Bash is an interpreter right?

Comment (vinjana) [reply RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy]: Indeed, it is. Maybe a service class to actually do this conversion from class fields to bash command line would be better.

Comment (dankwart-de) [reply RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy]: Ok, I'll approve it and will open an issue for this topic. Too big to do it now.

**Pull request merged**